shared/ui/utils/math-utils.js

```diff
@@ -108,10 +108,14 @@ export const MILLISECONDS_IN_A_SECOND = 1000
 export const MILLISECONDS_IN_A_MONTH = 2592000000
 export const WEEK = 604800000
 
-export function millisecondsToBlocks(start, end, blockDuration = 15) {
-  return Math.round((end - start) / MILLISECONDS_IN_A_SECOND/ blockDuration)
+export function millisecondsToBlocks(start, end, blockDuration = 15000) {
```

Comment:
Does this need to be changed before deploy?

Reply:
not technically, no, unless you want to implement a more sophisticated way to calculate the number of blocks